### DISCUSSION TITLE
Make private api calls within Vercel

### DISCUSSION THREAD

**Discussion by ilmoi** [vercel/community#557] [Help] [answered]
I have 2 apps deployed on vercel (separately but same account). 

Is there a way to restrict one of them to only accept requests from other apps within the same account?

In other words - is it possible to setup a private subnet between the two?

**Comment by swarnava** [comment, marked as answer]
Hi there,

Such restriction isn't possible in Vercel. However, you can always build a custom workflow in such way with authorization header that it only accepts and sends within your two apps.

**Reply by ilmoi** [reply]
Oh not good. The auth header approach doesn't work since I don't have a login system on the website. I guess moving to AWS is the only easy-ish option.
